xl2times/datatypes.py

```diff
@@ -84,7 +84,7 @@ class DataModule(str, Enum):
     trade = "SuppXLS/Trades/ScenTrade_*.*"
 
     @classmethod
-    def is_module_type(cls, path):
+    def _find_type_(cls, path: str) -> Enum | None:
```

Comment:
```suggestion
    def _find_type_(cls, path: str) -> DataModule | None:
```

Reply:
Oh if that doesn't pass the type checker because of the recursive reference to DataModule, I feel like I've read something that the solution is to use quotes around it? I can look it up later

Reply:
I'll try it out!

Reply:
It worked! I had to rename `_find_type_` though, because of `_sunder_ names, such as '_find_type_', are reserved for future Enum use` error.